### PR TITLE
More SQA tags.

### PR DESCRIPTION
**Pull request by jasondhales**
#13634

### CODE REVIEW

File: modules/combined/test/tests/cavity_pressure/tests [hunk 1]
```diff
@@ -21,6 +25,7 @@
     exodiff = 'cavity_pressure_rz_out.e'
     abs_zero = 1e-10
     recover = false
+    requirement = 'The system shall compute a cavity pressure based on the ideal gas law with an axisymmetric model'
```
Comment (gambka) [inline modules/combined/test/tests/cavity_pressure/tests]: with -> for

File: modules/combined/test/tests/internal_volume/tests [hunk 2]
```diff
@@ -111,50 +127,58 @@
...
     input = 'rz_cone.i'
     csvdiff = 'rz_cone_out.csv'
+    requirement = 'The system shall compute internal volume given an enclosing sideset for meshes with non axis-aligned geometries for solid_mechanics'
   [../]
 
   [./hex8]
     type = 'CSVDiff'
     input = 'hex8.i'
     csvdiff = 'hex8_out.csv'
+    requirement = 'The system shall compute internal volume given an enclosing sideset for 3D coordinates for solid_mechanics'
```
Comment (gambka) [inline modules/combined/test/tests/internal_volume/tests]: remove solid_mechanics

File: modules/combined/test/tests/internal_volume/tests [hunk 3]
```diff
@@ -111,50 +127,58 @@
...
     input = 'rz_quad8.i'
     csvdiff = 'rz_quad8_out.csv'
+    requirement = 'The system shall compute internal volume given an enclosing sideset for axisymmetric coordinates and quad8 elements'
   [../]
 
   [./rz_cone]
     type = 'CSVDiff'
     input = 'rz_cone.i'
     csvdiff = 'rz_cone_out.csv'
+    requirement = 'The system shall compute internal volume given an enclosing sideset for meshes with non axis-aligned geometries for solid_mechanics'
```
Comment (gambka) [inline modules/combined/test/tests/internal_volume/tests]: remove solid_mechanics

File: modules/combined/test/tests/internal_volume/tests [hunk 4]
```diff
@@ -28,13 +33,15 @@
     cli_args = 'Postprocessors/internalVolume/component=1 Outputs/file_base=rz2_sm_out'
     csvdiff = 'rz2_sm_out.csv'
     prereq = 'rz_sm'
+    requirement = 'The system shall compute internal volume given an enclosing sideset for axisymmetric coordinates and with the y coordinate for solid_mechanics'
```
Comment (gambka) [inline modules/combined/test/tests/internal_volume/tests]: Seems like we should say what the y-coordinate is being used for in the test.  Something like "... with the y coordinate used in the integration over the sideset..."?

File: modules/combined/test/tests/internal_volume/tests [hunk 5]
```diff
@@ -111,50 +127,58 @@
     cli_args = 'Postprocessors/internalVolume/component=1
                 Outputs/file_base=rz2_out'
     prereq = rz
+    requirement = 'The system shall compute internal volume given an enclosing sideset for axisymmetric coordinates and with the y coordinate'
```
Comment (gambka) [inline modules/combined/test/tests/internal_volume/tests]: Similar to the solid mechanics version of the test.

**Review by gambka — CHANGES_REQUESTED** [review]

File: modules/heat_conduction/doc/content/source/kernels/HomogenizedHeatConduction.md [hunk 6]
```diff
@@ -1,14 +1,27 @@
...
+\end{equation}
+and
+\begin{equation}
+k_{ij}^\text{H}=\frac{1}{\left|\text{Y}\right|}\int_\text{Y}k_{ij}\left(\bm{I} + \frac{\partial\psi^k}{\partial y_j}\right)\;\text{d}\bm{y}
+\end{equation}
+where $k^H_{ij}$ is the homogenized thermal conductivity.  See [cite](hales15homogenization).
+
+## Example Input File Syntax
+
+!listing modules/combined/test/tests/homogenization/heatConduction2D.i block=Kernels/HomogenizedHeatConduction
```
Comment (gambka) [inline modules/heat_conduction/doc/content/source/kernels/HomogenizedHeatConduction.md]: Changing `block=Kernels/HomogenizedHeatConduction` to either `block=Kernels/heat_rhs_x` or `block=Kernels/heat_rhs_y` should fix the failure.

File: modules/heat_conduction/doc/content/source/postprocessors/HomogenizedThermalConductivity.md [hunk 7]
```diff
@@ -1,14 +1,24 @@
...
+\end{equation}
+where $$k^H_{ij}$ is the homogenized thermal conductivity.  It is used in conjunction with the [Heat Conduction](HeatConduction.md) `Kernel` and the [Homogenized Heat Conduction](HomogenizedHeatConduction.md) `Kernel` to compute homogenized thermal conductivity values according to
+\begin{equation}
+\int_Y \frac{\partial v}{\partial y_i}k_{ik} \frac{\partial \psi^k}{\partial y_j} = \int_Y \frac{\partial v}{\partial y_i}k_{ik} \text{d}y
+\end{equation}
+where $k$ is the diffusion coefficient (thermal conductivity).  See [cite](hales15homogenization).
+
+## Example Input File Syntax
+
+!listing modules/combined/test/tests/homogenization/heatConduction2D.i block=Postprocessors/HomogenizedThermalConductivity
```
Comment (gambka) [inline modules/heat_conduction/doc/content/source/postprocessors/HomogenizedThermalConductivity.md]: `block=Postprocessors/k_xx` or `block=Postprocessors/k_yy` should fix the documentation error.

**Review by gambka — CHANGES_REQUESTED** [review]

File: modules/heat_conduction/doc/content/source/kernels/HomogenizedHeatConduction.md (hunk 6 above)
Comment (jasondhales) [reply modules/heat_conduction/doc/content/source/kernels/HomogenizedHeatConduction.md]: Thanks.

File: modules/heat_conduction/doc/content/source/postprocessors/HomogenizedThermalConductivity.md (hunk 7 above)
Comment (jasondhales) [reply modules/heat_conduction/doc/content/source/postprocessors/HomogenizedThermalConductivity.md]: Thanks.

File: modules/combined/test/tests/internal_volume/tests (hunk 5 above)
Comment (jasondhales) [reply modules/combined/test/tests/internal_volume/tests]: Done.

File: modules/combined/test/tests/internal_volume/tests (hunk 4 above)
Comment (jasondhales) [reply modules/combined/test/tests/internal_volume/tests]: Done.

File: modules/combined/test/tests/internal_volume/tests (hunk 3 above)
Comment (jasondhales) [reply modules/combined/test/tests/internal_volume/tests]: Done.

**Review by permcody — APPROVED** [review]

**Pull request merged**